src/main/resources/staff/templates/document/settlementAgreement.html

```diff
@@ -174,20 +256,40 @@ <h2>Settlement agreement</h2>
 
 <div class="section">
   <div class="section-heading">
-    <strong>Settlement agreement</strong>
+    <strong>What this agreement means</strong>
   </div>
   <div class="section-body">
     <table>
       <tr>
-        <td>Agreement:</td>
         <td>
-          {{ acceptedOffer }}
+          <p>This agreement settles the claim made by Jan Clark against Mary Richards. This includes all money owed in the claim, for example court fees, expenses or interest.</p>
```

Comment:
jan clark and mary richards ae?

Reply:
Ups, fast and furious changes, good spot.

Reply:
Done